tests/cypress/support/commands.js

```diff
@@ -28,34 +28,104 @@
 
 import '@testing-library/cypress/add-commands';
 
-Cypress.Commands.add('login', (username, password) => {
-	cy
-		.getCookies()
-		.then(cookies => {
-			let hasMatch = false;
-			cookies.forEach((cookie) => {
-				if (cookie.name.substr(0, 20) === 'wordpress_logged_in_') {
-					hasMatch = true;
-				}
-			});
-			if (!hasMatch) {
-				cy.visit('/wp-login.php').wait(1000);
-				cy.get('#user_login').type(username);
-				cy.get('#user_pass').type(`${ password }{enter}`);
+Cypress.Commands.add( 'login', ( username, password ) => {
+	cy.getCookies().then( ( cookies ) => {
+		let hasMatch = false;
+		cookies.forEach( ( cookie ) => {
+			if ( cookie.name.substr( 0, 20 ) === 'wordpress_logged_in_' ) {
+				hasMatch = true;
 			}
-		});
-});
-
-Cypress.Commands.add('logout', () => {
-	cy
-		.getCookies()
-		.then(
-			cookies => {
-				cookies.forEach(
-					cookie => {
-						cy.clearCookie(cookie.name);
-					}
-				)
+		} );
+		if ( ! hasMatch ) {
+			cy.visit( '/wp-login.php' ).wait( 1000 );
+			cy.get( '#user_login' ).type( username );
+			cy.get( '#user_pass' ).type( `${ password }{enter}` );
+		}
+	} );
+} );
+
+Cypress.Commands.add( 'logout', () => {
+	cy.getCookies().then( ( cookies ) => {
+		cookies.forEach( ( cookie ) => {
+			cy.clearCookie( cookie.name );
+		} );
+	} );
+} );
+
+Cypress.Commands.add( 'resetGeneralSettingTab', ( tableName ) => {
+	cy.task(
+		'queryDb',
+		`DELETE FROM
+  ${ tableName }
```

Comment:
Why are we passing the table name in here? Isn't it always going to be the same?

Reply:
If we are using the local then it would be same. I will change it